src/Hydrator/Strategy/NamingStrategy/NamingStrategyEnabledInterface.php

```diff
@@ -7,9 +7,9 @@
  * @license   http://framework.zend.com/license/new-bsd New BSD License
  */
 
-namespace Zend\Stdlib\Hydrator;
+namespace Zend\Stdlib\Hydrator\Strategy\NamingStrategy;
 
-use Zend\Stdlib\Hydrator\NamingStrategy\NamingStrategyInterface;
+use NamingStrategyInterface;
```

Comment:
no, NamingStrategyInterface already moved to same directory, or we can just remove the use statement